library/ix-dev/charts/nextcloud/questions.yaml

```diff
@@ -116,6 +136,16 @@ questions:
           schema:
             type: boolean
             default: false
+        - variable: max_upload_size
+          label: "Max Upload Size (Giga Bytes)"
+          description: |
+            Applies the timeout to the following settings:</br>
+            - client_max_body_size in nginx</br>
+            - post_max_size and upload_max_filesize in php</br>
+          schema:
+            type: int
```

Comment:
Should we have a minimum here and above?

Reply:
Agreed